src/app/invest/invest.component.ts

```diff
@@ -129,7 +129,7 @@ export class InvestComponent implements OnInit {
         );
     }
 
-    backToSingleOfferScreen(uuid: string) {
-        this.router.navigate([`/dash/offers/${uuid}`]);
+    backToSingleOfferScreen() {
+        this.router.navigate(['../'], {relativeTo: this.route});
```

Comment:
This wasn't working anymore (I don't know why). @mightymatth could it be that adding coop to URL broke this?